docs/development.rst

```diff
@@ -40,8 +125,8 @@ mss repository needs a different folder, e.g. workspace/mss. Avoid to mix data a
 
 Some of used packages are in the conda-forge channel located, so we have to add this channel to the default::
 
-  $ conda config --add channels conda-forge
-  $ conda config --add channels defaults
+$ conda config --add channels conda-forge
```

Comment:
You need to add some indentation or similar (verbatim text) to keep this from becoming a normal paragraph.